test/jsonrpc-examples/add-module.error.response

```diff
@@ -0,0 +1 @@
+{"jsonrpc":"2.0","id":4,"error":{"code":2,"data":{"context":null,"error":"FOUNDRY-MAIN-DEPENDS-MODULE: Duplicate module"},"message":"Could not verify pattern"}}
```

Comment:
This was a missing test case, and it hit a pattern match error in the `decodeKoreRpc` function